tests/Unit/MutationTest.php

```diff
@@ -169,4 +286,19 @@ public function testCustomValidationErrorMessages(): void
             $this->assertEquals($messages->first('test_with_rules_non_nullable_input_object.nest.email'), 'The test with rules non nullable input object.nest.email must be a valid email address.');
         }
     }
+
+    public function testRuleCallbackArgumentsMatchesTheInput(): void
+    {
+        $field = new UpdateExampleMutationWithInputType2;
+        $attributes = $field->getAttributes();
+
+        try {
+            $attributes['resolve'](null, [
+                'test_with_rules_callback_params' => [
+                    'otherValue' => 1337,
+                ],
+            ], [], null);
+        } catch (ValidationError $e) {
+        }
```

Comment:
If we don't care about the exceptions details (beyond the message), I recommend using the built-in phpunit-way: `$this->expectException(…);`